src/util/settings/connection_client.cc

```diff
@@ -213,14 +214,16 @@ QString serverScriptDefault() {
   server_script = qApp->applicationDirPath() + "/../veles-server/srv.py";
 #elif defined(Q_OS_LINUX)
   server_script = qApp->applicationDirPath() + "/../share/veles-server/srv.py";
-#else
+#elif defined(Q_OS_MAC)
   server_script = qApp->applicationDirPath() + "/../Resources/veles-server/srv.py";
+#else
+  #error OS not supported
```

Comment:
IMO it's ok in other places. E.g.: in the file you linked to, the if checks whether we are working on Windows and then does something special, otherwise does something else. No reason to list specific OSes there, as it's only there to support Windows as a special case.